pkg/minikube/problem/err_map.go

```diff
@@ -56,7 +56,7 @@ var vmProblems = map[string]match{
 		Issues: []int{1926, 4206},
 	},
 	"HYPERKIT_NOT_FOUND": {
-		Regexp:         re(`Driver "hyperkit" not found. Do you have the plugin binary .* accessible in your PATH?`),
```

Comment:
unrelated change?

Reply:
yeah but the change actually is good.

Reply:
that regex on master wont match the error